wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java

```diff
@@ -108,6 +109,12 @@
   static final ClassName NULLABLE = ClassName.get("androidx.annotation", "Nullable");
   static final ClassName CREATOR = ClassName.get("android.os", "Parcelable", "Creator");
 
+  static final String REDACTED_TOSTRING_ASSIGN = "\", $N=██\"";
+  static final int REDACTED_TOSTRING_ASSIGN_LENGTH = REDACTED_TOSTRING_ASSIGN.length() - 2;
+
+  static final String TOSTRING_ASSIGN = "\", $N=\"";
+  static final int TOSTRING_ASSIGN_LENGTH = TOSTRING_ASSIGN.length() - 2;
```

Comment:
inline the escaped quotes back at the use sites, avoiding the need to offset by 2 here?

Reply:
It's not the quotes I'm subtracting, it's the `$N`, which I should perhaps add a comment for.